app/models/translation_override.rb

```diff
@@ -45,14 +45,18 @@ class TranslationOverride < ActiveRecord::Base
 
   validate :check_interpolation_keys
 
+  enum :status, %i[up_to_date outdated invalid_interpolation_keys]
```

Comment:
This is only ever set by the background job, and are used to indicate whether old overrides have gone stale due to updates in core.